src/components/OrganisationPublicView/HorizontalTabs/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { selectedOrganisationPageTab } from 'actions/organisation';
+import clsx from 'clsx';
+import { FC, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import styles from './index.module.scss';
+
+const tabs = [
+  { id: 0, text: 'overview' },
+  { id: 1, text: 'projects' },
+  { id: 2, text: 'people' },
+  { id: 3, text: 'jobs' },
+];
+
+interface TabItem {
+  id: number;
+  text: string;
+}
+
+interface IHorizontalTabs {
+  customButtonLabel: string;
+  customButtonLink: string;
+}
+
+const HorizontalTabs: FC<IHorizontalTabs> = ({
+  customButtonLabel,
+  customButtonLink,
+}) => {
+  const dispatch = useDispatch();
+  const [buttonSelected, setButtonSelected] = useState(0);
+
+  const handleButtonClick = (tabItem: TabItem) => {
+    setButtonSelected(tabItem.id);
```

Comment:
Don't think we need local state if we are using the same thing in redux store also.